homeassistant/components/foscam/__init__.py

```diff
@@ -14,12 +14,6 @@
 PLATFORMS = ["camera"]
 
 
-async def async_setup(hass: HomeAssistant, config: dict):
-    """Set up the foscam component."""
```

Comment:
I thought this method was needed even with config_entries, to signal that the integration is ready ?

Reply:
This is no longer needed since a month or so.

Reply:
Thanks good to know.